app/services/customers/upsert_from_api_service.rb

```diff
@@ -145,7 +151,7 @@ def call
 
     private
 
-    attr_reader :organization, :params
+    attr_reader :billing_entity, :organization, :params
```

Comment:
shall we define belling_entity as an instance variable?
and also we can then define it on init...